corehq/apps/locations/dbaccessors.py

```diff
@@ -92,3 +92,23 @@ def user_ids_at_accessible_locations(domain_name, user):
     from corehq.apps.locations.models import SQLLocation
     accessible_location_ids = SQLLocation.active_objects.accessible_location_ids(domain_name, user)
     return user_ids_at_locations(accessible_location_ids)
+
+
+def assigned_location_by_user_id(domain, location_ids):
```

Comment:
hmm, ok. yeah i'm not sure we want to open it up to all assigned locations because they also won't necessarily be in the list of supervisor child locations so we might not have a name for them. i think it's fine to only use primary location